### ISSUE TITLE
how to align Mandarin dataset by mfa？

### ISSUE THREAD

**Issue by yileld** [ming024/FastSpeech2#59]
the Mandarin lexicon.txt in your project seems different from mfa pretrained model
and it will failed to align，saying "There were phones in the dictionary that do not have acoustic models"
![image](https://user-images.githubusercontent.com/13084050/117618251-3ced4b00-b1a0-11eb-8654-54c7699ee456.png)

**Comment by ming024** [comment]
@yileld If you want to use your other lexicon, just put it in ``lexicon/``, and change the symbols in ``text/pinyin.py``.

**Comment by yileld** [comment]
> @yileld If you want to use your other lexicon, just put it in `lexicon/`, and change the symbols in `text/pinyin.py`.

nonono I didnt mean to change lexicon，I just want to know how you get Mandarin textgrid by your lexicon because it seems mismatch mfa pretrained model，do you use "mfa train xxx" command?

**Comment by ming024** [comment]
@yileld yeah I trained the MFA model from scratch since I wish to handle the er-hua （ㄦ話） in the Chinese language carefully, while this feature is not provided in the pretrained MFA model.

**Comment by hertz-pj** [comment]
> @yileld yeah I trained the MFA model from scratch since I wish to handle the er-hua （ㄦ話） in the Chinese language carefully, while this feature is not provided in the pretrained MFA model.

Do you have any plan to share MFA model with you trained to handle the er-hua~

**Comment by yuxiazff** [comment]
> > @yileld yeah I trained the MFA model from scratch since I wish to handle the er-hua （ㄦ話） in the Chinese language carefully, while this feature is not provided in the pretrained MFA model.
> 
> Do you have any plan to share MFA model with you trained to handle the er-hua~

Could you share the mfa acoustic model of AISHELL3，thank you very much!

**Comment by yuxiazff** [comment]
> > @yileld yeah I trained the MFA model from scratch since I wish to handle the er-hua （ㄦ話） in the Chinese language carefully, while this feature is not provided in the pretrained MFA model.
> 
> Do you have any plan to share MFA model with you trained to handle the er-hua~

@ming024 Could you share the mfa acoustic model of AISHELL3，thank you very much!
